tests/foreman/ui_airgun/test_discoveredhost.py

```diff
@@ -0,0 +1,414 @@
+        changed for multiple hosts
+
+    :BZ: 1634728
+
+    :CaseLevel: Integration
+
+    :CaseImportance: High
+    """
+    host_names = [create_discovered_host()['name'] for _ in range(2)]
+    new_org = entities.Organization().create()
+    module_loc.organization.append(new_org)
+    module_loc.update(['organization'])
+    new_loc = entities.Location(organization=[module_org]).create()
+    with session:
+        session.organization.select(org_name=module_org.name)
+        session.location.select(loc_name=module_loc.name)
+        values = session.discoveredhosts.search(
+            'name = "{0}" or name = "{1}"'.format(*host_names)
+        )
+        assert set(host_names) == {value['Name'] for value in values}
+        session.discoveredhosts.apply_action(
+            'Assign Organization',
+            host_names,
+            values=dict(organization=new_org.name)
+        )
+        assert not session.discoveredhosts.search(
+            'name = "{0}" or name = "{1}"'.format(*host_names)
+        )
+        session.organization.select(org_name=new_org.name)
+        values = session.discoveredhosts.search(
+            'name = "{0}" or name = "{1}"'.format(*host_names)
+        )
+        assert set(host_names) == {value['Name'] for value in values}
+        session.discoveredhosts.apply_action(
+            'Assign Location',
+            host_names,
+            values=dict(location=new_loc.name)
+        )
+        assert not session.discoveredhosts.search(
+            'name = "{0}" or name = "{1}"'.format(*host_names)
+        )
+        session.location.select(loc_name=new_loc.name)
+        values = session.discoveredhosts.search(
+            'name = "{0}" or name = "{1}"'.format(*host_names)
+        )
+        assert set(host_names) == {value['Name'] for value in values}
+
+
+@skip_if_not_set('compute_resources', 'vlan_networking')
+@tier3
+def test_positive_reboot(session, module_org, module_loc, provisioning_env):
+    """Reboot a discovered host.
+
+    :id: 5edc6831-bfc8-4e69-9029-b4c0caa3ee32
+
+    :Setup: Host should already be discovered
+
+    :expectedresults: Host should be successfully rebooted.
+
+    :CaseLevel: System
+    """
+    with LibvirtGuest() as pxe_host:
+        host_name = pxe_host.guest_name
+        with session:
+            session.organization.select(org_name=module_org.name)
+            session.location.select(loc_name=module_loc.name)
+            discovered_host_values = session.discoveredhosts.wait_for_entity(
+                host_name)
+            assert discovered_host_values['Name'] == host_name
+            host_ip = discovered_host_values['IP Address']
+            assert host_ip
+            # Ensure that the host is reachable
+            assert _is_host_reachable(host_ip)
+            session.discoveredhosts.apply_action('Reboot', host_name)
+            # Ensure that the host is not reachable
+            assert not _is_host_reachable(host_ip, expect_reachable=False)
```

Comment:
emm. Shouldn't we wait till host become reachable to actually verify 'Host should be successfully rebooted.'?

Reply:
Think it's sufficient to check that the command passed and the host became unreachable (as the original version do), but if you want we can wait for that host to became  reachable again.

Reply:
But unfortunately after some experiment seems the host is completely deleted from the infra after reboot perhaps this the infra configuration to keep only the running vms:
so if our VM name macb68e5d3620db.XXXX.com

before reboot
```
virsh # list
 Id    Name                           State
----------------------------------------------------
 13869 macb68e5d3620db.XXXX.com running
```

just after reboot command from sat

```
virsh # list
 Id    Name                           State
----------------------------------------------------
 13869 macb68e5d3620db.XXXX.com shut off
```

after this step the VM completely disappear from the VMs list

any way we are not testing the libvirt and abaility to do reboot , but that the command was transmitted to infra.

after reboot the VM also fail to be deleted as does not exist any more
from robottelo log
```
virsh destroy macb68e5d3620db.XXXX.com
rror: Failed to destroy domain macb68e5d3620db.satellite.lab.eng.rdu2.redhat.com
error: Requested operation is not valid: domain is not running
```

Reply:
Think no as per investigations we made  with @san7ket it seems that the VM is shutdown it's not deleted it simply not listed as running and cannot reboot as it has not been provisioned, the boot loader must be configured at provisioning stage and as no provisioning took place, there was no boot media to boot from.

Reply:
so the VM after reboot was intended to boot from it's own media, but that media was not provisioned.